### ISSUE TITLE
USWDS - Bug: Accessibility - Multiple unnamed banner landmarks

### ISSUE THREAD

**Issue by jaredcunha** [uswds/uswds-site#2226]
### Describe the bug

Both the `Banner` and `Header` components contain `<header>` elements, which show up in the rotor as banner landmark. It might not be clear to users with screen readers the difference between the two banner regions.

### Steps to reproduce the bug

1. Got to the USWDS homepage
2. Turn of VoiceOver
3. Pull up the rotor menu (VO+U)
4. Use the arrow keys to pull up the Landmarks
5. Notice two unnamed banners.

### Expected Behavior

One or or both of the components container a banner region should have an accessible name.

### Screenshots

<img width="699" alt="Screenshot 2023-08-02 at 7 23 40 PM" src="https://github.com/uswds/uswds/assets/1094951/c7ea4f45-55ce-467d-bd87-b8bafda6a948">

### System setup

- Current USWDS
- MacOS
- VoiceOver

### Additional context

Some additional can be found in [Scott O'Hara's article on landmarks](https://www.scottohara.me/blog/2018/03/03/landmarks.html).

> Since there is no native “banner element” in HTML, modern browsers are meant to treat the `<header>`, that is the nearest direct child of the `<body>` element, as a banner landmark. While there can be multiple `<header>` elements used within a web document, ideally there should only be a single banner landmark exposed. **Instances where multiple banners may be desired would be if there were nested document or application roles within a single view. In such a case, it would be best to ensure each of these were given their own accessible name, as this would allow assistive technologies to jump to each nested banner and have a better idea of which “banner” they had navigated to.**

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by mejiaj** [comment]
Thanks for reporting this issue! I've moved it to the site repo because:
1. USWDS header & banner **don't** have `role="banner"`.
2. Header guidance is outdated and mentions `role="banner"` - [header accessibility guidance](https://designsystem.digital.gov/components/header/#:~:text=The%20Design%20System%20uses%20semantic%20header%20and%20nav%20elements%20with%20role%3D%22banner%22%20and%20role%3D%22navigation%22%20respectively.%20role%3D%22banner%22%20is%20your%20masthead.).
![image](https://github.com/uswds/uswds-site/assets/3385219/9a635f9a-c469-47ad-ab2e-86237d3b915b)

**Comment by mejiaj** [comment]
@jaredcunha thanks for the follow-up information. I saw that the site had older markup and perhaps was too hasty in moving it here. 

---

@amyleadem thanks for outlining those next steps. Once we go through confirmation we should create GH issues to track them.
